game/Enums.h

```diff
@@ -86,6 +86,7 @@ enum UnitClassifactions			// [Note: **Typo** in name]
 	clsNotSet = 0x10,			// 10
 	clsAll = 0x11,				// 11 All vehicles and buildings
 };
+
 typedef UnitClassifactions UnitClassifications;		// Typo fixing typedef
 // Note: clsAttack applies to all attack vehicles with the following weapons:
 //		Microwave
```

Comment:
Hmm, we should probably rework the documentation here. The comment that follows pertains to the `enum` above. It should probably be moved above the `enum`, and maybe restructured to flow nicely. The `typedef` is strongly related to the `enum`, though doesn't really belong between the definition and related documentation.

Can optionally be done as a separate branch.

Reply:
Opened PR #31